src/components/SignName.vue

```diff
@@ -138,7 +149,7 @@ export default {
 
     methods: {
         addressCheck () {
-          this.errorMessages = this.address && !this.name
+          this.errorMessages = this.address && !this.first_name && !this.last_name
```

Comment:
Same thing here about the `firstName` and `lastName` values.

Reply:
Updated firstName and lastName values to match Vue instance variables as they are called in data